ocdskingfishercolab/__init__.py

```diff
@@ -269,7 +204,7 @@ def download_data_as_json(data, filename):
     files.download(filename)
 
 
-def download_package_from_query(sql, params=None, package_type=None):
+def download_package_from_ipython_sql_results(results, package_type=None):
```

Comment:
I'm not sure about this change. Per https://github.com/open-contracting/kingfisher-colab/pull/36/files#r532282006, I think analysts will mostly set `autopandas` to `True` so they won't have a result set to pass to this function.

Reply:
I've changed this back, e.g. https://colab.research.google.com/drive/1cUYY4on72831DPSiQ_JLxJEY2uGTfVrN#scrollTo=bdYdAYo08ou0

I've still removed params, because the easiest implementation was to pull the params from the cell context, as the magic does. Hopefully this makes sense to users, as it's consistent with how the magic behaves.